bundle/edu.gemini.ags/src/main/java/edu/gemini/ags/gems/GemsGuideStarSearchOptions.java

```diff
@@ -112,19 +112,12 @@ public GemsTipTiltMode getGemsTipTiltMode() {
         }
     }
 
+    private final GemsInstrument instrument;
+    private final GemsTipTiltMode tipTiltMode;
+    private final Set<Angle> posAngles;
 
-    public static final String DEFAULT_CATALOG = CatalogChoice.DEFAULT.catalogName();
-
-    private String opticalCatalog = DEFAULT_CATALOG;
-    private String nirCatalog = DEFAULT_CATALOG;
-    private GemsInstrument instrument;
-    private GemsTipTiltMode tipTiltMode;
-    private Set<Angle> posAngles = new HashSet<>();
-
-    public GemsGuideStarSearchOptions(final String opticalCatalog, final String nirCatalog, final GemsInstrument instrument,
+    public GemsGuideStarSearchOptions(final GemsInstrument instrument,
                                       final GemsTipTiltMode tipTiltMode, final Set<Angle> posAngles) {
```

Comment:
Yes, we would like to support PPMXL (which goes fainter) for December.  In the not-to-distant future we will want to support new catalogs from Pann_STARRS and LSST.